app/components/ui/power-select-multiple/options.hbs

```diff
@@ -0,0 +1,38 @@
+<ul 
+  role="listbox" 
+  {{did-insert this.addHandlers}}
+  {{will-destroy this.removeHandlers}} 
+  ...attributes
+>
+  {{#if @select.loading}}
+    {{#if @loadingMessage}}
```

Comment:
I think it'll be better if we either use an `or` helper or have a getter to compute these 2 values together instead of nesting ifs